llvm/lib/Transforms/IPO/OpenMPOpt.cpp

```diff
@@ -1507,6 +1510,40 @@ struct OpenMPOpt {
     return Changed;
   }
 
+  /// Tries to remove known runtime symbols that are optional.
+  bool removeRuntimeSymbols() {
```

Comment:
It's technically not "remove", right? Would that be more like `privatizeRpcClientGlobal` or so?

Reply:
I was thinking we might want something for another global in the future. I updated the comment to make it clearer that  'remove' is referring to the symbol table.

Reply:
WFM